index.html

```diff
@@ -0,0 +1,263 @@
+					<div>10 poumds $40.00</div>
+					<div>20 pounds $70.00</div>
+						
+				</section>
+
+			</article>
+			<article class="salescard">
+			<header class="scheader">
+				<h1>Janky Cheese</h1>
+			</header>
+
+				<section class="section1">
+					<div class="pic">
+						<img class="displayed" src="http://www.gourmetegypt.com/media/catalog/product/cache/1/image/1000x/9df78eab33525d08d6e5fb8d27136e95/m/i/mimolette-old-cheese.jpg" height="60" width="60">
+					</div>
+			
+					<div class="description">
+					Doner turkey ground round, bresaola strip steak ham sausage. Venison ham tri-tip andouille salami. Ground round pork chop alcatra turkey swine capicola jowl rump pork belly filet mignon turducken short loin fatback pancetta biltong. Tenderloin pancetta bacon, shankle cupim prosciutto strip steak pork loin leberkas jerky ball tip meatloaf corned beef ham hock chicken.
+				
+					</div>
+
+
+					<div class="availability">
+					available
+				
+					</div>
+				</section>
+				
+
+				<section class="specs">
+						<div>
+					    <h1 class="spechead"> Specifications </h1>
+					    </div>
+				
+
+						<div class="spec-item">
+					    Size: 12x16
+
+						</div>
+						<div class="spec-item"> 
+					    Weight: 5 lbs
+
+						</div>
+
+
+					<footer class="foot"> &copy 2016 ya mama's house  </footer>
+				</section>
+				<section class="pricing">
+					<h1> Pricing </h1>
+					<div>5 pounds  $15.00</div>
+					<div>10 poumds $28.00</div>
+					<div>20 pounds $40.00</div>
+						
+				</section>
+
+			</article>
+			<article class="salescard">
+			<header class="scheader">
+				<h1>Lanky Cheese</h1>
+			</header>
+
+				<section class="section1">
+					<div class="pic">
+						<img class="displayed" src="https://thumbs.dreamstime.com/z/turkish-old-cheese-aged-kashar-35260294.jpg" height="60" width="60">
+					</div>
+			
+					<div class="description">Doner turkey ground round, bresaola strip steak ham sausage. Venison ham tri-tip andouille salami. Ground round pork chop alcatra turkey swine capicola jowl rump pork belly filet mignon turducken short loin fatback pancetta biltong. Tenderloin pancetta bacon, shankle cupim prosciutto strip steak pork loin leberkas jerky ball tip meatloaf corned beef ham hock chicken.
+				
+					
+					</div>
+
+
+					<div class="availability">
+					available
+				
+					</div>
+				</section>
+				
+
+				<section class="specs">
+						<div>
+					    <h1 class="spechead"> Specifications </h1>
+					    </div>
+
+				
+
+						<div class="spec-item">
+					    Size: 14x14	
+
+						</div>
+						<div class="spec-item"> 
+					    Weight: 3 lbs
+						</div>
+
+
+					<footer class="foot"> &copy; 2016 ya mama's house  </footer>
+				</section>
+				<section class="pricing">
+					<h1> Pricing </h1>
+					<div>5 pounds  $21.00</div>
+					<div>10 poumds $38.00</div>
+					<div>20 pounds $70.00</div>
+						
+				</section>
+
+			</article>
+				
+			
+
+			
+</body>
+</html>'/n'
```

Comment:
I see what you did there, but just press enter after the `</html>` or use the sublime setting to do it for you. Either way, remove the `'/n'`